1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,151 @@
+Feature: User edits Project
+
+    @xxx
+    Scenario: User sees the Project Wizard
+        Given User is logged in
+        When they open the Project Wizard
+        Then the User is the Project Owner
```

Comment:
Leser dette som at den som åpner Project Wizard skal bli satt til owner, men det stemmer sikkert ikke? Tenker da hvis man åpner wizard for å endre ett eksisterende prosjekt

Reply:
hvis denne bare skal beskrive opprettelse av nytt prosjekt:
```
Given User opens My Projects
When they click Create New Project Button
Then they see tabs:
(....)
```

Reply:
Har justert